hepcrawl/spiders/t2k_spider.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015, 2016 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Spider for T2K."""
+
+from __future__ import absolute_import, print_function
+
+from urlparse import urljoin
+
+from scrapy import Request
+from scrapy.spiders import XMLFeedSpider
+
+from ..items import HEPRecord
+from ..loaders import HEPLoader
+from ..utils import split_fullname
+
+
+class T2kSpider(XMLFeedSpider):
+
+    """T2K crawler
+    Scrapes theses metadata from T2K experiment web page.
+    http://www.t2k.org/docs/thesis
+
+    1. `parse_node` will get thesis title, author and date from the listing.
+    2. If link to the splash page exists, `scrape_for_pdf` will try to fetch
+       the pdf link and possibly also the abstract.
+    3. `build_item` will build the HEPRecord.
+
+    Example usage:
+    scrapy crawl t2k
+    scrapy crawl t2k -a source_file=file://`pwd`/tests/responses/t2k/test_list.html -s "JSON_OUTPUT_DIR=tmp/"
```

Comment:
Aah ok, I see. The url http://www.t2k.org/docs/thesis/001 is correct. So the file name should be `001`, not `001.html`. Should this single file be added to MANIFEST? I modified everything else to work with the correct name.

Also, should I (re)add the `test_list.html`? It's the whole web page http://www.t2k.org/docs/thesis. It's not really needed anywhere as the tests are using the `001` file.